packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -312,10 +326,15 @@ options:
     opts.dartTestOut = results['dart_test_out'];
     opts.objcHeaderOut = results['objc_header_out'];
     opts.objcSourceOut = results['objc_source_out'];
-    opts.objcOptions.prefix = results['objc_prefix'];
+    opts.objcOptions = ObjcOptions(
+      prefix: results['objc_prefix'],
+    );
     opts.javaOut = results['java_out'];
-    opts.javaOptions.package = results['java_package'];
-    opts.dartOptions.isNullSafe = results['dart_null_safety'];
+    opts.javaOptions = JavaOptions(
+      className: 'Pigeon',
```

Comment:
We should be generating a classname that matches the name of the file that is getting generated (ie Foobar.java makes class Foobar).  That code should be somewhere here.  If we are overwriting it somewhere else we should probably leave classname nullable so we can get an error if it isn't set, instead of generating with some placeholder name.

Reply:
Removed the 'Pigeon' literal